studio/lib/payloads/compensations.ts

```diff
@@ -8,6 +8,12 @@ export interface Benefit {
   richText: PortableTextBlock[];
 }
 
+export interface BenefitsGroup {
+  _key: string;
+  location: string;
+  benefitsGroup: Benefit[];
```

Comment:
Hm, the interface is named `BenefitsGroup`  and has a ´benefitsGroup´ a but it's an interface for the  `benefitsByLocation`. I think we can update this a little. 🤔

Reply:
I don't have a good answer right this second, but feel free to adjust schema names as well if say `benefitsGroup` doesn't cut it

Initially I'd think

```
export interface BenefitsByLocation{
    _key: string;
    _type: string;
    location: string;
    benefits: Benefit[];
}
```